pkg/server/commands/write.go

```diff
@@ -137,7 +139,7 @@ func (c *WriteCommand) validateNoDuplicatesAndCorrectSize(deletes []*openfgav1.T
 
 func handleError(err error) error {
 	if errors.Is(err, storage.ErrTransactionalWriteFailed) {
```

Comment:
can you update the text of this error, too?

https://github.com/openfga/openfga/blob/aa0c8162003a389263ee27c6be6731773bda0147/pkg/storage/errors.go#L18

to "transactional write failed due to conflict"